expyfun/visual/_visual.py

```diff
@@ -826,6 +826,76 @@ def __init__(self, ec, colors=('w', 'k')):
         self.set_radius(1, 1, units='pix')
 
 
+class ProgressBar(object):
+    """A progress bar that can be displayed between sections
+
+    This uses two rectangles, one outline, and one solid to show how much
+    progress has been made in the experiment.
+
+    Parameters
+    ----------
+    ec : instance of ExperimentController
+        Parent EC.
+    pos : array-like
+        4-element array-like with X, Y center and width, height.
```

Comment:
this could be clearer: `4-element array-like (x, y, width, height) where x and y are coordinates of the box center.`

Reply:
Do you want me to change this for Diamond, Rectangle, and FixationDot as well? I agree that it's confusing but assumed it was just me.

Reply:
Sure